lib/sidekiq/scheduled.rb

```diff
@@ -147,13 +147,16 @@ def random_poll_interval
         # As we run more processes, the scheduling interval average will approach an even spread
         # between 0 and poll interval so we don't need this artifical boost.
         #
-        if process_count < 10
+        count = process_count
+        interval = poll_interval_average(count)
```

Comment:
I added the process count argument so we don't have to do `scard` multiple times in a single poll.